frontend/src/components/Overview/CommunityNews.vue

```diff
@@ -28,17 +52,53 @@
         </b-card-body>
       </b-card>
     </div>
+    --></b-card>
   </div>
 </template>
 <script>
 import News from '@/assets/News/news.json'
+
+// Berechne den heutigen Tag des Jahres
+const jetzt = new Date()
+const start = new Date(jetzt.getFullYear(), 0, 0)
```

Comment:
Is loaded one time, so if there is a lot of time until the click on the calculate button this could be wrong!